src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -943,6 +943,8 @@ namespace cryptonote
       uint64_t height;
       uint64_t target_height;
       uint64_t difficulty;
+      std::string wide_difficulty;
+      bool difficulty_overflow;
```

Comment:
Why not use `uint64_t difficulty_top64;` as done in cryptonote_protocol_handler?

Reply:
Because I think this is simpler. Old code still works now (until diff overflows), new code use uses the new one. Do you think the other one is simpler ? I can't recall the reason why I added a top64 in protocol now, there was a good reason to do with compatibility IIRC...

Reply:
One way or the other, whichever is deemed more useful, should be used IMO. I don't see a point in using two different styles in different parts of the codebase.

Reply:
You're right, overflow is simpler as top64 since it can also serve as a flag anyway.